lib/matplotlib/axes/_base.py

```diff
@@ -2512,6 +2517,50 @@ def handle_single_axis(scale, autoscaleon, shared_axes, interval,
     def _get_axis_list(self):
         return (self.xaxis, self.yaxis)
 
+    def _update_title_position(self, renderer):
+        """
+        Update the title position based on the bounding box enclosing
+        all the ticklabels and x-axis spine and xlabel...
+        """
+        _log.debug('update_title_pos')
+
+        if self._autotitlepos is not None and not self._autotitlepos:
+            _log.debug('title position was updated manually, not adjusting')
+            return
+
+        titles = (self.title, self._left_title, self._right_title)
+
+        if self._autotitlepos is None:
+            for title in titles:
+                x, y = title.get_position()
+                if not np.isclose(y, 1.0):
```

Comment:
Very dumb question incoming :sheep:... With this implementation, there is no way to pin the title y-position at 1.0 and still avoid the auto title positioning, is there not? But in case anybody would like to do so (for whichever good or bad reason ^^), I guess that a reasonable workaround would be to pin the title y-position at something like 1.001, right?

Reply:
Thats correct.  I *could* do something like find the first call that sets the title at 1.0 and then flag future calls to `set_position` as being "pinned", even if it is y=1.0. But setting to 1.01 will give the same result.

Reply:
IMHO, setting `y=1.001` is a good enough solution. Maybe it should be put in the “what's new” entry (just for the workaround to be documented somewhere)?

Reply:
Done!